Makefile

```diff
@@ -10,15 +10,17 @@ env:
 	virtualenv --python=python3 env
 	$(IN_ENV) pip install --upgrade -r requirements.txt
 
-.PHONY: env
-
-
-format: ${PYTHON_SRCS}
-	$(IN_ENV) yapf -i $?
+format: env ${PYTHON_SRCS}
```

Comment:
Adding `env` like this will result in the env command be invoked repeatedly.  I don't think this is the desired behavior?

Reply:
Ok, I've fixed that.